_posts/2020-06-14-Emendations to Source Editions on IrishGen.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+date: 2020-06-14 14:00:00 +0100
+title: Emendations to Source Editions on IrishGen
+author: et
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+## Introduction
+When adding genealogies to IrishGen, as Christopher Yocum has [already noted](https://cyocum.github.io/2020/06/07/Human-Curation-and-Digital-Datasets.html), we generally use existing printed editions rather than making our own transcriptions. This is due both to limitations on our time and to the need to remain accountable when encoding data. The printed editions we use need to be based on a single manuscript, as we want to be able to accurately encode data each manuscript and thus track all the possible variants in the tradition. Fortunately, several medieval genealogical collections are available in such editions and these are available on [CELT](https://celt.ucc.ie/), including those in the [Book of Leinster](https://celt.ucc.ie//published/G800011F/index.html), [Rawl. B.502](https://celt.ucc.ie//published/G105003/index.html), and [Laud Misc. 610](https://celt.ucc.ie//published/G105005/index.html).
+
+Occasionally, however, we encounter incongruities in existing editions and clarify the situation by checking online images of the manuscript. These images are available via repositories like [ISOS](https://www.isos.dias.ie) or [Digital.Bodleian](https://digital.bodleian.ox.ac.uk/). If the edition turns out to be in serious error, we follow the original manuscript in our encoding. This blogpost describes one such eventuality.
+
+## The Anachronistic Ancestor of Dál nAraide
+Cenél Lóegaire were a minor southern Uí Néill dynasty based around modern-day [Trim](https://www.logainm.ie/1416699.aspx ), Co. Meath. The genealogists traced their ancestry back to [Lóegaire](https://doi.org/10.1093/ref:odnb/15872), son of Níall Noígiallach, the pagan king of Tara whose druids confronted St. Patrick.
```

Comment:
I know this story has been published somewhere. Do you have a link to that edition or do you think this is common knowledge?

Reply:
It appears in both Muirchiu and Tirechan and later Patrician hagiography. There is an independent prose narrative version too. I think all these are referenced in the ODNB article, though, aren't they?

Reply:
Yes, you are right. I missed the link.